lib/pdk/util/vendored_file.rb

```diff
@@ -51,8 +51,11 @@ def download_file
         uri = URI.parse(url)
         http = Net::HTTP.new(uri.host, uri.port)
         http.use_ssl = true
-        # TODO: Get rid of this
-        http.verify_mode = OpenSSL::SSL::VERIFY_NONE if Gem.win_platform?
+        if Gem.win_platform?
+          cert_path = 'C:/Program Files/Puppet Labs/DevelopmentKit\ssl\cert.pem'
+          http.cert = OpenSSL::X509::Certificate.new(cert_path)
```

Comment:
@david22swan I think you want to call the `ca_file=` setter, passing the path to `ca.pem` file. The method you're calling is for the client cert which shouldn't be necessary for the pdk. See https://docs.ruby-lang.org/en/master/Net/HTTP.html#attribute-i-ca_file